Explorer/Assets/DCL/Infrastructure/ECS/SceneLifeCycle/IncreasingRadius/SceneLoadingLimit.cs

```diff
@@ -0,0 +1,21 @@
+using Decentraland.Kernel.Comms.Rfc4;
+
+namespace ECS.SceneLifeCycle.IncreasingRadius
+{
+    //Struct that will be used to dynamically modify the maximum amount of scenes that can load
+    //in the memory system
+    public struct SceneLoadingLimit
```

Comment:
It probably should be a class unless you want to query it, otherwise you won't be able to set values from outside (I suppose it's better to split responsibilities and introduce it in another system)

Reply:
Cool, changing it to a class. Next PR will change the values dynamically